src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
all stars modifiers should work (like on hw with stars)

Reply:
and try to use [cycle](https://sass-lang.com/documentation/at-rules/control/for) for it